packages/gluestick/src/config/webpack/webpack.config.server.js

```diff
@@ -71,5 +78,32 @@ module.exports = (
       'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
     }),
   );
-  return serverConfiguration(config, settings);
+  config.plugins.push(
+    new webpack.optimize.LimitChunkCountPlugin({
+      maxChunks: 1,
+    }),
+  );
+  config.output = {
```

Comment:
Very temporary, this is making up for the removal of universal-webpack. Will figure out what to do along with consumer-frontend testing.